### DISCUSSION TITLE
convert won't read bitmap font

### DISCUSSION THREAD

**Discussion by kresp0** [ImageMagick/ImageMagick#5796] [Help] [answered]
Hi,

I'm trying to use a bitmap font with convert, but it fails. Tried the gunzipped file and still fails. 

I can open the ic8x8u.pcf.gz file with FontForge. Also tried the bdf version of the font, with the same results.

The font files are on the same directory I'm running convert from.

Do you have any suggestions? Thanks!

```
$ echo "test" | convert -size 160x24 -append xc:"#e0f8cf" -font ic8x8u.pcf.gz -fill "#071821" -annotate +8+16 "@-" test.png
convert-im6.q16: unable to read font `ic8x8u.pcf.gz' @ error/annotate.c/RenderFreetype/1481.

```

**Comment by urban-warrior** [comment, marked as answer]
Post a link to `ic8x8u.pcf` so we can download it and reproduce the issue.

**Reply by kresp0** [reply]
You can download the font here:
[https://cartobase.es/ic8x8u.pcf](https://cartobase.es/ic8x8u.pcf)

**Reply by urban-warrior** [reply]
Currently ImageMagick only supports scaleable fonts.   PCF is fixed-sized.  You may be able to convert the older-style fixed-sized font to a more modern scalable font with FontForge.

**Reply by kresp0** [reply]
Thanks. Found a couple of scalable fonts that work well.
